integration/test_number_of_indexes_on_replica_fullsync.py

```
@@ -0,0 +1,39 @@
+"""
+Verify a replica reports the correct search_number_of_indexes after a full sync.
+"""
+
+import pytest
+from valkeytestframework.conftest import resource_port_tracker
+from valkey_search_test_case import ValkeySearchTestCaseBase
+from indexes import *
```

Ruff (0.15.21)

[error] `from indexes import *` used; unable to detect undefined names

(F403)